src/Uno.UI/UI/Xaml/DependencyPropertiesImplementation.tt

```diff
@@ -72,8 +72,8 @@ namespace <#= @namespace.Name #>
 
         protected virtual void On<#= dp.Name #>Changed(<#= dp.PropertyType #> old<#= dp.Name #>, <#= dp.PropertyType #> new<#= dp.Name #>)
         {
-            On<#= dp.Name #>ChangedPartial(old<#= dp.Name #>, new<#= dp.Name #>);
             On<#= dp.Name #>ChangedPartialNative(old<#= dp.Name #>, new<#= dp.Name #>);
+            On<#= dp.Name #>ChangedPartial(old<#= dp.Name #>, new<#= dp.Name #>);
```

Comment:
I don't like change at all, but I'm wondering if it will break anything. The logic of Popup.IsOpenChanged is scattered between these 2 methods in an unnecessarily complex way and it could definitely use a refactoring. This is not the first time I encountered this.